packages/compiled-assets/src/index.ts

```diff
@@ -220,25 +296,73 @@ async function buildAssets(sourceDirectory: string, buildDirectory: string) {
     entryNames: '[dir]/[name]-[hash]',
     outbase: sourceDirectory,
     outdir: buildDirectory,
+    metafile: true, // Write metadata about the build
+  });
+
+  // For now, we only build ESM bundles for scripts that are split into chunks (i.e. Preact components)
+  // Using 'type=module' in the script tag for ESM means that it is loaded after all 'classic' scripts, which causes issues with bootstrap-table. See https://github.com/PrairieLearn/PrairieLearn/pull/12180.
```

Comment:
Nit, we try to hard-wrap comments so that they don't go much past 80-100 characters.

Reply:
Done in [7d1fe99](https://github.com/PrairieLearn/PrairieLearn/pull/12157/commits/7d1fe99032df1a89fafd5c85a1009b0c440fa359).